### PR TITLE
Adding TestingPays

### PR DESCRIPTION
**Pull request by James-Byrne**
Please review our [Contribution Guidelines](https://github.com/doomspork/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.

Review your changes and complete the checklist below.

- [X] Company name
- [X] Company description
- [X] Company location
- [X] GitHub link
- [X] How Elixir is used
- [X] Sorted alphabetically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +232,8 @@ their hiring process - and grow faster. Amsterdam, Netherlands.
 
 * [Spreedly](https://www.spreedly.com) ([Github](https://github.com/spreedly)) - Spreedly believes the best payment experience derives from excellent software combined with business flexibility. Durham, NC, USA.
 
+* [TestingPays](https://testingpays.com) ([GitHub](https://github.com/TestingPays/)) - TestingPays makes integrating and testing your payment systems easier by providing tips, guides and allowing you to simulate payment gateway responses. We power our apps in browser live tail using elixir channels (replacing ActionCable). We are based in Dublin Ireland.
```
Comment (doomspork) [inline README.md]: Howdy @James-Byrne!  Could you please remove the whitespace between these two listings?  None of the others have it 👍

**Comment by James-Byrne** [comment]
Hi @doomspork! Thanks for the quick response, thats updated now

**Comment by James-Byrne** [comment]
@doomspork Just noticed the https issue. Seems we have it enabled on the [web app](https://admin.testingpays.com/) but not the front for the site. We'll try and get that sorted this week.

**Comment by doomspork** [comment]
No worries @James-Byrne, thanks for the note 👍

**Pull request merged**